home/tests/test_assessment_import_export.py

```diff
@@ -0,0 +1,512 @@
+import csv
+import re
+from collections.abc import Callable, Iterable
+from dataclasses import dataclass
+from functools import wraps
+from io import BytesIO, StringIO
+from pathlib import Path
+from queue import Queue
+from typing import Any
+
+import pytest
+from openpyxl import load_workbook
+from pytest_django.fixtures import SettingsWrapper
+from wagtail.models import Locale  # type: ignore
+
+from home.assessment_import_export import import_assessment
+from home.import_assessment_pages import ImportAssessmentException
+from home.models import (
+    HomePage,
+)
+
+from .page_builder import (
+    MBlk,
+    MBody,
+    PageBuilder,
+    WABlk,
+    WABody,
+)
+
+IMP_EXP_DATA_BASE = Path("home/tests/import-export-data")
+
+ExpDict = dict[str, Any]
+ExpPair = tuple[ExpDict, ExpDict]
+ExpDicts = Iterable[ExpDict]
+ExpDictsPair = tuple[ExpDicts, ExpDicts]
+
+
+def filter_both(
+    filter_func: Callable[[ExpDict], ExpDict]
+) -> Callable[[ExpDict, ExpDict], ExpPair]:
+    @wraps(filter_func)
+    def ff(src: ExpDict, dst: ExpDict) -> ExpPair:
+        return filter_func(src), filter_func(dst)
+
+    return ff
+
+
+@filter_both
+def strip_leading_whitespace(entry: ExpDict) -> ExpDict:
+    # FIXME: Do we expect imported content to have leading spaces removed?
+    bodies = {k: v.lstrip(" ") for k, v in entry.items() if k.endswith("_body")}
```

Comment:
We don't have any fields that end with `_body` for assessments, so it seems like this won't do anything, so we can probably remove it from the assessment tests?